src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```
@@ -0,0 +1,28 @@
+﻿<!--
+***********************************************************************************************
+Microsoft.Common.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

@rainersigwald I would appreciate your review for this file.

---

I don't think you need tools version, it doesn't do much here.

---

```suggestion
<Project>
```

Neither of these is required any more.